indico/htdocs/js/indico/modules/categories/management.js

```diff
@@ -59,6 +56,18 @@
             });
         });
 
+        $bulkMoveButton.on('click', function(evt) {
+            evt.preventDefault();
+            if ($(this).hasClass('disabled')) {
```

Comment:
What I mean is that in both `L62` and `L65` you create 2 identical jQuery objects and you could avoid that by creating the `var $this`. `L65` still belongs to the body of the function.